tcms/testcases/models.py

```diff
@@ -355,7 +355,7 @@ def remove_cc(self, email_addrs):
             email_addrs = [email_addrs]
         for address in email_addrs:
             if address in emailaddr_list:
-                emailaddr_list.remove(address)
+                emailaddr_list.remove(address)#pylint:disable=Protected-access
```

Comment:
Dos this have anything to do with silencing the `form-field-help-text-used` ? It is not even in the right location.